share/completions/pygmentize.fish

```diff
@@ -1,8 +1,12 @@
 function __fish_print_pygmentize
-    set -l lines (pygmentize -L $argv[1] | string match -r '^(?:\* |    ).*(?:)$' | string replace -r '\* (.*):$' '$1' | string replace -r '^(.*).$' '$1' | string trim)
+    set -l lines (pygmentize -L $argv[1] | string match -r '^(?:\* |    ).*(?:)$' | string replace -r '\* (.*):$' '$1' | string replace -r '^(.*)^\.' '$1' | string trim)
```

Comment:
I might not be reading the regex right here. Is that "A line, and then the next line has a '.'"?

What does this output look like?